docs/reference/worker-configuration-reference.md

```diff
@@ -149,6 +149,24 @@ core:
   noPublish: true
 ```
 
+### core.noOwnerRefs
+
+Setting `core.noOwnerRefs` to `true` disables setting the owner references
+to Pod for NodeFeature object.
```

Comment:
Thank you @ozhuraki for the improvement! Just one nit about the wording (we don't just mark to pod as the owner but also the parent owner of the pod). E.g.

```suggestion
Setting `core.noOwnerRefs` to `true` disables setting the owner references
of the NodeFeature object created by the nfd-worker.
```

Ditto the cmdline reference